openshift-ci/run_e2e.sh

```diff
@@ -5,8 +5,10 @@ metallb_dir="$(dirname $(readlink -f $0))"
 source ${metallb_dir}/common.sh
 
 METALLB_REPO=${METALLB_REPO:-"https://github.com/openshift/metallb.git"}
-BGP_TYPE=${BGP_TYPE:-""}
-IP_STACK=${IP_STACK:-""}
+export BGP_TYPE=${BGP_TYPE:-""}
```

Comment:
Why do you export that?
I presume to be picked up by another bash script that runs within that script.
My question is it possible instead of export to be args? That will make better the dependencies from one scrip tto another?

Reply:
it's being used by the other scripts, yes. Changing to args, although there are still some envs that are coming from outside we can't do much about that (plus here we are adding a feature, not refactoring)